unittests/snapshot_tests.cpp

```diff
@@ -697,13 +701,17 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(test_kv_snapshot, SNAPSHOT_SUITE, snapshot_suites)
             auto snapshot = SNAPSHOT_SUITE::finalize(writer);
 
             auto cfg = chain.get_config();
-            cfg.use_rocksdb_for_disk = rocks_load;
+            if (rocks_load) {
+               cfg.backing_store = eosio::chain::backing_store_type::ROCKSDB;
+            } else {
+               cfg.backing_store = eosio::chain::backing_store_type::CHAINBASE;
+            }
             // create a new child at this snapshot
             sub_testers.emplace_back(cfg, SNAPSHOT_SUITE::get_reader(snapshot), generation);
 
             // increment the test contract
             signed_transaction trx;
-            trx.actions.push_back({{{N(snapshot), N(active)}}, N(snapshot), N(eosio.kvram), {}});
+            //trx.actions.push_back({{{N(snapshot), N(active)}}, N(snapshot), N(eosio.kvram), {}});
```

Comment:
remove the next line (with kvdisk) and un-comment this line

Reply:
Will do.